src/main/java/org/plumelib/util/ArraysPlume.java

```diff
@@ -4121,15 +4121,22 @@ boolean isPartitioningFor(List<T> elts) {
      * @param elt the element to add
      * @return a new partitioning just like this one, but with elt added to the ith part
      */
+    @SuppressWarnings("index:argument")
```

Comment:
Every `@SuppressWarnings` must have a comment that explains why the warning that it suppresses is a false positive.